app/views/layouts/dc_repository_about.html.erb

```diff
@@ -0,0 +1,33 @@
+<%# OVERRIDE: Hyrax v3.4.2 - add classes for custom design of dc repository theme %>
```

Comment:
Which layout is this overriding?  Could you note that in the comment?

Reply:
I'm guessing this is, https://github.com/samvera/hyrax/blob/64c0bbf0dc0d3e1b49f040b50ea70d177cc9d8f6/app/views/layouts/hyrax/dashboard.html.erb

Please move line 4 to the top of the document.  Especially before the line 3 which is an HTML comment declared before we declare the DOCTYPE.

This may not be necessary but having that DOCTYPE at the beginning of the file is a good plan when considering HTML parsers and what have you.

Reply:
This is a custom partial. Comment has been updated.